Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -730,6 +752,40 @@ private void SetUpClippingPrimitive()
             }
         }
 
+        // Setting up a collider and a near interaction touchable to allow interaction with empty spaces like corners or in between children items
+        private void SetupScrollingBackground()
+        {
+            if (scrollingBackground == null)
+            {
+                scrollingBackground = GetComponent<BoxCollider>();
+                if (scrollingBackground == null)
```

Comment:
nit: because I just read this https://docs.unity3d.com/ScriptReference/Component.TryGetComponent.html - we could use:
if (!TryGetComponent(scrollingBackground))